### PR TITLE
MAX-5331 React UI cannot be rendered via index.html

### PR DESCRIPTION
**Pull request by benjaminliugang**
The root cause of this issue is that we missed the index html template in product config file. Just added.

And we also need to manually add css import when use containUIComponents case.

@joshsylvester @shinxi , please help review.

### CODE REVIEW

File: packages/react-scripts/template/public/index.html [hunk 1]
```diff
@@ -56,8 +59,8 @@
         }, false);
       })();
     </script>
-    <script src="/bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
-    <link rel="import" href="/bower_components/px-theme/px-theme-styles.html">
+    <script src="%PUBLIC_URL%/bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
```
Comment (joshsylvester) [inline packages/react-scripts/template/public/index.html]: CRA can't assume that we're using bower - in the future (and for other teams) this won't always use predix components, or even bower. can we remove this dependency, or only add it if we're using predix?

Comment (benjaminliugang) [reply packages/react-scripts/template/public/index.html]: We already added `<% if (htmlWebpackPlugin.options.containUIComponents) { %>` this check, it means only when we use ui-components-predix with bower then we will add this dependency.

`const containUIComponents = fs.existsSync(path.resolve(paths.appNodeModules, "@svmx/ui-components-predix/bower_components"));`

**Comment by shinxi** [comment]
Changes looks good to me. But pls also add the url predix at [ui-components-predix/withReactWrapper.js](https://github.com/ServiceMax-Engineering/ui-components-predix/blob/master/lib/polymer/withReactWrapper.js#L11).

**Comment by benjaminliugang** [comment]
Added prefix by https://github.com/ServiceMax-Engineering/ui-components-predix/pull/73. Please review.

File: packages/react-scripts/template/public/index.html (hunk 1 above)
Comment (joshsylvester) [reply packages/react-scripts/template/public/index.html]: sorry, i missed the if statement above (possibly because it wasn't indented)

**Pull request merged**